.github/push.yml

```
@@ -0,0 +1,31 @@
+name: Push Workflow for develop-postgres
```

yamllint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)